surfer/viz.py

```diff
@@ -965,28 +965,26 @@ def add_data(self, array, min=None, max=None, thresh=None,
                  vertices=None, smoothing_steps=20, time=None,
                  time_label="time index=%d", colorbar=True,
                  hemi=None, remove_existing=False, time_label_size=14,
-                 initial_time=None):
+                 initial_time=None, scale_factor=None, vector_alpha=None):
         """Display data from a numpy array on the surface.
 
-        This provides a similar interface to add_overlay, but it displays
+        This provides a similar interface to
+        :meth:`surfer.viz.Brain.add_overlay`, but it displays
         it with a single colormap. It offers more flexibility over the
-        colormap, and provides a way to display four dimensional data
-        (i.e. a timecourse).
+        colormap, and provides a way to display four-dimensional data
+        (i.e., a timecourse) or five-dimensional data (i.e., a
+        vector-valued timecourse).
```

Comment:
Isn't the vector data strictly speaking 7 dimensional, or 6 if you discount magnitude? :)